src/pcsc.rs

```diff
@@ -40,36 +40,22 @@ impl Transport for PcscTransport {
         let rapdu = transport.transmit_apdu(applet_select_apdu)?;
         let status_response = StatusResponse::from_cbor(rapdu.to_vec())?;
 
-        // get common fields
```

Comment:
Is there a reason to duplicate these lines below instead of leaving here?

Reply:
I think it would be cleaner to unpack the status response within the card types constructor.  I've been primarily working with the tapsigner, so that was the only constructor I added. This doesn't need to be in this PR, or if it is I should update for all three card types

Reply:
Ah ok I see now this is related to your `TapSigner::from_status` change. No need to change it.